### ISSUE TITLE
Batch Dice

### ISSUE THREAD

**Issue by TheOrgg** [buttonmen-dev/buttonmen#2661]
I mentioned Batch Dice on the forums, and a few discussions came up about them.

https://buttonmen.fandom.com/wiki/Monstrous_Musicians

Batch Dice are groups of dice you select to use at the beginning of the game.  Looking into the "We're the de-vaccumnators of rules" and the discussion that came up in the forum thread, I'd like to propose that the Dice Batch is chosen before Select Dice are chosen, and both of those before Swing Dice are set.  This would allow the most flexibility out of what people could do with the Batch Dice.

I'd also like to suggest it be possible to have more than two Batches, as I see potential for this type of dice grouping.

From my talks with one of the people behind the Bridge and Tunnel set and the intended random nature of "Steve," I also believe that Batch Dice should work with Mood Swing notation, randomly switching between games from among the Dice Batches available to the die.

**Comment by TheOrgg** [comment]
(I realize this is probably a 'Pony' issue, but I have insomnia again.  Might as well use it for something.)

**Comment by TheOrgg** [comment]
@blackshadowshade , I know this one's probably a long ways off, but what would you recommend be the shorthand for writing the different batches?  I've been using [Recipe // Recipe] as shorthand, and putting the Mood Swing outside the last square bracket.  

I'd like to format it to where it is probably useable when you finally get around to coding it.  I could see replacing the // with ; or another symbol, for example.  If not for Steve in Bridge and Tunnel, I'd say drop the [ 's from them, but if that happens, how would you notate the mood swing.

If you don't want to dedicate any brainpower to this at the moment, understood.  I'd just like to know how best to write it on the wiki.

**Comment by blackshadowshade** [comment]
I guess that we've got the issue of keeping the notation for these separate to option, plasma, select, and other skills.

We could go with angle brackets and pipes, e.g.:

⟨ (2) (8) (8) (8) (12) | (1) (4) (4) (4) (6) ⟩

**Comment by TheOrgg** [comment]
Cool.  How do you make angle brackets, as I assume they're different than Less than and Greater than?  (and the pipe is this one, | , yes?  The "Absolute Value" notation?

**Comment by blackshadowshade** [comment]
Yeah, I don't know. I copied the angle brackets off the web, knowing them from my days with LaTeX. And pipe is just the standard one on the keyboard.

**Comment by blackshadowshade** [comment]
According to https://en.wikipedia.org/wiki/Bracket#Encoding, I was thinking of

U+27E8 | Mathematical left angle bracket | &lang;

and

U+27E9 | Mathematical right angle bracket | &rang;

**Comment by TheOrgg** [comment]
‹ ›  Nope, not those...

Character Map's giving me a "hole" icon when I try those.  Guess it'll have to be "find, copy and paste" whenever I need those, then.  At least the pipe is the |AbVal| one on the keyboard.
